src/ml/neural_net/style_transfer/mps_style_transfer.h

```diff
@@ -0,0 +1,34 @@
+/* Copyright © 2018 Apple Inc. All rights reserved.
```

Comment:
May as well use the current date in new headers

Reply:
Done for all my files!